cirq-core/cirq/ops/clifford_gate.py

```diff
@@ -43,6 +43,36 @@ def _to_pauli_transform(matrix: np.ndarray) -> Optional[PauliTransform]:
     return None
 
 
+def _to_clifford_tableau(
+    rotation_map: Optional[Dict[Pauli, PauliTransform]] = None,
+    *,
+    x_to: Optional[PauliTransform] = None,
+    z_to: Optional[PauliTransform] = None,
+) -> qis.CliffordTableau:
+    """Transfer the rotation map to clifford tableau representation"""
+    if x_to is None and z_to is None and rotation_map is None:
+        # coverage: ignore
+        raise ValueError(
```

Comment:
Let's cover this with tests instead of ignoring it, error scenarios are as important as happy path scenarios.

Reply:
Done.